### DISCUSSION TITLE
pnpm purge ?

### DISCUSSION THREAD

**Discussion by aruniverse** [pnpm/pnpm#4297] [Q&A]
Hi! I typically use pnpm via a wrapper, rush, and theres a useful command `rush purge` to completely nuke the node_modules. Right now i'm trying to test out using pnpm directly without rush. I didnt see any equivalent in pnpm directly. Any recommendations besides implementing it myself? Plans to add something similar like this directly to pnpm?

**Comment by gluxon** [comment]
+1. Not essential, but something that removes `node_modules` from packages matching `pnpm-workspace.yaml` would be cool. I'm currently just doing `rm -rf node_modules && packages/*/node_modules`.
